util/find-doc-nits

```diff
@@ -151,17 +151,17 @@ sub name_synopsis()
     }
 }
 
-# Check if SECTION is located before BEFORE
+# Check if SECTION ($3) is located before BEFORE ($4)
```

Comment:
How can something be located before "BEFORE"?  Shouldn't it be located before "AFTER"?

_Never mind, forget my comment. Just picking nits on the nit-picker. 😜_